microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -6,69 +6,119 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 05/16/2023
+ms.date: 06/02/2023
 ---
 # Data and privacy in WebView2
 
-The WebView2 team tracks data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. This article is for WebView2 developers.
-WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License. Additionally, WebView2 follows the standards outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).
+The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.
 
-In general, WebView2 data collection goes through the same strict standards as Microsoft Edge data collection. The main telemetry sources are Chromium and Microsoft Edge telemetry infrastructure, Windows data reporting, and Watson (Microsoft Internal) infrastructure to collect crash dumps.
-Any WebView2 data collection complies with the laws, regulations, and best practices for privacy, and WebView2 has mechanisms to ensure privacy.
+To view the WebView2 Runtime Terms and Conditions License, go to the [Microsoft.Web.WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2) NuGet package page, and then click the **License Info** link.  For example, [WebView2/1.0.1774.30/License](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License).

```

Comment:
@nishitha-burman @maurawinstanley 
Is this link & paragraph what we mean by "the WebView2 Runtime Terms and Conditions License"?
Is the name "the WebView2 Runtime Terms and Conditions License" correct?

https://www.nuget.org/packages/Microsoft.Web.WebView2
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License

Reply:
An open Issue from 2021 is about a different License that does say "terms" and "WebView2":  at page https://developer.microsoft.com/en-us/microsoft-edge/webview2/ 
in the "Evergreen Runtime Installer" section, click the "x64" button.  Opens a dialog in the browser:  

![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/45198fce-0823-414f-a2f1-a729170da83a)

Reply:
outdated, see comment https://github.com/MicrosoftDocs/edge-developer/pull/2594/files#r1215028818